pygmt/session_management.py

```diff
@@ -15,6 +15,7 @@ def begin():
     prefix = "pygmt-session"
     with Session() as lib:
         lib.call_module("begin", prefix)
+        lib.call_module("set", "GMT_COMPATIBILITY 6")
 
 
 def end():
```

Comment:
Can't find a good way to reset the `GMT_COMPATIBILITY` setting back to what's in the original `gmt.conf`. Probably doesn't matter too much?

Reply:
See https://github.com/GenericMappingTools/pygmt/issues/428#issuecomment-629809995